shell/probe/raindrop.ps1

```diff
@@ -1,10 +1,10 @@
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
 
-function Execute { 
-    Param([String]$File)
+function Execute {
+    Param([String]$File, [String]$StdoutFile, [String]$StderrFile)
 
     try {
-        $R = (Start-Process -FilePath $File -Wait -NoNewWindow -PassThru).ExitCode
+        $R = (Start-Process -FilePath $File -Wait -NoNewWindow -PassThru -RedirectStandardOutput $StdoutFile -RedirectStandardError $StderrFile).ExitCode
```

Comment:
We should see if we can redirect to the same file, so we can keep ordering? but maybe windows doesn't let you have the same file open for writing in two places.

Reply:
sad day
```
Start-Process : This command cannot be run because "RedirectStandardOutput" and "RedirectStandardError" are same. Give
different inputs and Run your command again.
```